Sources/MembraneRTC/Types/Endpoint.swift

```diff
@@ -3,49 +3,59 @@ public struct Endpoint: Codable {
     public let type: String
     public let metadata: Metadata
     public let trackIdToMetadata: [String: Metadata]?
+    public let tracks: [String: TracksAddedEvent.Data.TrackData]?
```

Comment:
wouldn't it be easier if we had no `?` here? it can be always an empty array if there are no tracks?

Reply:
i will ask if tracks can be null to JF backend devs